components/profile-controller/controllers/profile_controller.go

```diff
@@ -442,6 +442,9 @@ func (r *ProfileReconciler) getAuthorizationPolicy(profileIns *profilev1.Profile
 								"/healthz",
 								"/metrics",
 								"/wait-for-drain",
+								"/ready",
+								"/v1/models/*",
```

Comment:
@surajkota allowing paths does not lock down the traffic from other namespaces. I think we still want the changes in https://github.com/kubeflow/kubeflow/pull/6013/files for only allowing requests from `knative-serving` namespace and in addition we need to lock down the original namespace of the request, unfortunately I couldn't find a header to check the original namespace when the request proxyed by activator and the closest one I found is the header `k-original-host` but that requires creating the authorization policy per service level instead of namespace level.

Reply:
Maybe we should rely on kubeflow-userid header?